firebase-common/src/test/java/com/google/firebase/heartbeatinfo/HeartBeatInfoStorageTest.java

```diff
@@ -75,6 +75,30 @@ public void storeOneHeartbeat_storesProperly() {
     assertThat(results.size()).isEqualTo(0);
   }
 
+  @Config(sdk = 29)
+  @Test
+  public void storeOneHeartbeat_updatesProperly() {
```

Comment:
can we tests with several dates? new user agent in a different date doesn't override the old user agent